query_sheet.rb

```diff
@@ -547,10 +546,12 @@ def formatted_candidates_to_import(candidates)
         } #{
           export_candidate['last_name']
         }",
-        'Type' => ARGV[1] || '2020',
+        'Type' => type,
         'RYBID' => export_candidate['nationbuilder_id'],
         'Phone' => format_phone_number(export_candidate),
         'Email' => export_candidate['email'],
+        'Status' => '=if(indirect("a"&row())="","",if(indirect("p"&row())="Declined","D",if(indirect("cb"&row())="yes","RECD",if(indirect("ca"&row())="Done","P/DONE",if(indirect("ca"&row())="Moving Along","P/OK",if(indirect("ca"&row())="Started","P/S",if(indirect("ca"&row())="Struggling","P/STRG",if(indirect("ca"&row())="Not Started","P/NS",if(indirect("by"&row())="yes","P/PKU",if(indirect("bw"&row())="yes","P/PLAN",if(indirect("bv"&row())="Attempt #3","P/3C",if(indirect("bv"&row())="Attempt #2","P/2C",if(indirect("bv"&row())="Attempt #1","P/1C",if(indirect("br"&row())>0,if(indirect("p"&row())="Yes",if(indirect("bv"&row())="No Attempts","P/NC",if(indirect("u"&row())="Complete",if(indirect("br"&row())>1,"INT/E+","INT/E"),"CONF/E"),if(indirect("p"&row())="Maybe","MAY/E",if(indirect("o"&row())="Attempt #3","3C/E",if(O2="Attempt #2","2C/E",if(indirect("o"&row())="Attempt #1","1C/E","NC/E")))))),if(indirect("u"&row())="Complete",if(indirect("bv"&row())="No Attempts","P/NC","INT"),if(indirect("p"&row())="Yes","CONF",if(indirect("p"&row())="Maybe","MAY",if(indirect("o"&row())="Attempt #3","3C",if(indirect("o"&row())="Attempt #2","2C",if(indirect("o"&row())="Attempt #1","1C","NC"))))))))))))))))))))',
+        'Events' => '=if(AQ4="yes",1,0)+if(AV4="yes",1,0)+if(BA4="yes",1,0)+if(BF4="yes",1,0)+if(BK4="yes",1,0)+if(BP4="yes",1,0)+if(AG4="yes",1,0)+if(AL4="yes",1,0)',
```

Comment:
Will the use of "4" in the formula here cause problems? Should it be &row() instead, like for Status?